projects/core/utils/format/format-phone.ts

```diff
@@ -1,6 +1,7 @@
 import {CHAR_PLUS} from '@taiga-ui/cdk';
 
 /**
+ * TODO: Guys, should we mark this function as deprecated?
```

Comment:
Sure, just make sure to add example on how to use new maskitoTransform in jsdocs